### DISCUSSION TITLE
Pointwise numerical dissipation using WENO

### DISCUSSION THREAD

**Discussion by j-ostiguy** [CliMA/Oceananigans.jl#3479] [Computational science!]
Hi,

I would like to calculate the numerical dissipation of the WENO scheme. I've been running my simulation with order 9 as an implicit LES but would like to calculate the numerical dissipation to use epsilon in a gas exchange parameterization. I've been using Oceanostics to calculate the energy budget terms and the residual. My idea has been to calculate the advection with a second and more accurate scheme to estimate the numerical dissipation, but I'm not quite sure if this is a good method with the ILES. 

@glwagner mentioned that @simone-silvestri has been working on this, and could maybe provide some guidance.

Thank you!

**Comment by simone-silvestri** [comment]
Hello @j-ostiguy. The research is not yet completed, but we are implementing the method described in this paper https://arxiv.org/abs/2310.16768 (the author of the paper is leading the project).

Unfortunately using another scheme will not work. We probably do not have a scheme with a lower truncation error than WENO 9, so if you subtract the results of another scheme you will add the truncation error of the other scheme (you could try with WENO 11 but most likely the scheme is already converged at order 9).

**Reply by glwagner** [reply]
@simone-silvestri can we test the method? Is there code?
